src/stores/linkStore.ts

```diff
@@ -75,11 +97,22 @@ export class LinkStore {
         };
     }
 
+    @action
+    public setStartMarkerCoordinates = (startMarkerCoordinates: LatLng | null) => {
+        this._startMarkerCoordinates = startMarkerCoordinates;
+    }
+
+    @action
+    public setExistingTransitTypes = (usedTransitTypes: TransitType[]) => {
```

Comment:
This variable is probably not required to be in the store?
Actually this variable could be equally well a normal local variable in the component. Since, there is no need for this variable to be observable. We just set and read it.